### PR TITLE
#161962689 User should be able to see all his/her bookmarked articles

### PR DESCRIPTION
**Pull request by ucheg6**
#### What does this PR do?
- create an endpoint that enables a user see all his/her bookmarked articles
#### Description of Task to be completed?
- write a function that returns all bookmarked articles of a user
- create an endpoint that gets all bookmarked articles of a user
- write tests for all functionalities
#### How should this be manually tested?
- Git clone the repository
- cd into the `haven-ah-backend` folder
- run `npm run start-dev`
- Fire up the route `localhost:5000/api/v1/users/articles/bookmarks` as a get request
- To test the entire application, run `npm run test`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161962689](https://www.pivotaltracker.com/story/show/161962689)
#### Screenshots (if appropriate)
screen shot 2018-11-16 at 5 44 34 pm
screen shot 2018-11-16 at 9 47 35 am

#### Questions:
N/A

### CODE REVIEW

**Comment by victor-nweze-deel** [comment]
Looks good.

File: server/repository/bookmarkRepository.js [hunk 1]
```diff
@@ -21,6 +21,33 @@ class BookmarkRepository {
...
     return bookmark;
   }
+
+  /**
+   * Function to get bookmarked article by a user
+   * @param { integer } userId
+   * @returns {object} bookmark object
+   ** otherwise it throws an error
+   */
+  static async getBookmarkedArticles(userId) {
```
Comment (TheoOkafor) [inline server/repository/bookmarkRepository.js:31]: Nice work @ucheg6

**Review by TheoOkafor — COMMENTED** [review]
This looks good.

**Comment by seyi-adeleke** [comment]
use `bookmarks` in place of `rows`

**Review by seyi-adeleke — APPROVED** [review]

**Comment by ucheg6** [comment]
@seyi-adeleke I have changed rows to bookmarks

**Comment by seyi-adeleke** [comment]
whats the new api endpoint?

**Comment by ucheg6** [comment]
> whats the new api endpoint?

@seyi-adeleke we resolved to use `api/v1/users/:username/bookmarks`

**Pull request merged**